### ISSUE TITLE
Incorrect `postMessage` signature for `DedicatedWorkerGlobalScope`

### ISSUE THREAD

**Issue by jespertheend** [mdn/content#29346]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/DedicatedWorkerGlobalScope/postMessage

### What information was incorrect, unhelpful, or incomplete?

There's an incorrect `postMessage(options)` signature listed

### What did you expect to see?

```
postMessage(message)
postMessage(message, transfer)
postMessage(message, options)
```

### Do you have any supporting links, references, or citations?

https://html.spec.whatwg.org/multipage/workers.html#dedicated-workers-and-the-dedicatedworkerglobalscope-interface lists the following two signatures:

- `postMessage(any message, sequence<object> transfer);`
- `postMessage(any message, optional StructuredSerializeOptions options = {});`

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/api/dedicatedworkerglobalscope/postmessage`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/DedicatedWorkerGlobalScope/postMessage
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/dedicatedworkerglobalscope/postmessage/index.md
* Last commit: https://github.com/mdn/content/commit/9cc36c338a51ee47665abb59829e0b7ca3ae44b4
* Document last modified: 2023-09-21T08:51:41.000Z

</details>

**Comment by Josh-Cena** [comment]
cc @skyclouds2001

**Comment by hamishwillee** [comment]
Good catch. Should be fixed by #29349

**Comment by skyclouds2001** [comment]
Thanks for this issue!
